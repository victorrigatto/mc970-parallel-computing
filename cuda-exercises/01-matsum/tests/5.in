12100
14853
