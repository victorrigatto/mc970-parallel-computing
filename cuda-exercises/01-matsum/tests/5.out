4843668756300
